src/extension/provider/statefulAuth.ts

```diff
@@ -248,6 +237,71 @@ export class StatefulAuthProvider implements AuthenticationProvider, Disposable
     this.#disposables.forEach((d) => d.dispose())
   }
 
+  public async loadFromSecrets() {
+    const secretsFile = '/etc/secrets/playground-dev'
```

Comment:
It should be overwritable by a setting. `/etc...` does not sound on non-Linux file systems. In other instances, we use the extension directory available in `ExtensionContext` to ensure it is always readable, irrespective of where the extension runs. The dir name should reflect what its purpose is and not its application. The word playground is irrelevant here. Something like `./secerts/authToken` is a good choice.

We should have a separate setting (default true) to toggle whether or not the token should be deleted once used.